integration-tests/README.md

```diff
@@ -6,7 +6,7 @@ See also: [Bueller](Bueller.md) - service for invoking tests
 
 1. Start by making sure you're running the correct versions of Node and Yarn, as described in the [root README](../README.md).
 2. Run `yarn install` in this directory.
-3. To set up an environment, run `node scripts/initializeEnvironment.js`.
+3. To set up an environment (e.g., dev, alpha, prod. Not your local environment), run `node scripts/initializeEnvironment.js`.
```

Comment:
I would extract this into its whole own section.  I think there should be one section for "here are instructions for you to set up your local environment to be able to run the integration tests" and a separate section for "here's what you need to do to configure a live Terra environment to be used by the integration test suite".

Reply:
Done!